frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -0,0 +1,108 @@
+import { execSync } from 'child_process';
+import * as _ from 'lodash';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { click } from '@console/shared/src/test-utils/utils';
+import { isNodeReady } from '@console/shared/src/selectors/node';
+import {
+  addCapacityLbl,
+  addCapacityBtn,
+  ocsOp,
+  storageClusterView,
+  storageClusterRow,
+} from '../views/add-capacity.view';
+
+const NAMESPACE = 'openshift-storage';
```

Comment:
With the ocs install in, we have few these constants defined there.
Please import from there.

Reply:
@afreen23 can you point me where its defined

Reply:
You can get this here : `frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts`

Reply:
https://github.com/bipuladh/console/blob/5d4a52a156ca90f05d2ad093919debfcd258eda2/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts#L2

Reply:
ack